### PR TITLE
Deadly skills

### PR DESCRIPTION
**Pull request by sunchron**
Pull request within a repository.

### CODE REVIEW

**Comment by sunchron** [comment]
backup to master

File: introduction.md [hunk 1]
```diff
@@ -1 +1,4 @@
-I'm introducing myself to the group.  My github is sunchron.  My favorite Code School path is javascript.
\ No newline at end of file
+I'm introducing myself to the group.  My github is sunchron.  My favorite Code School path is javascript. My deadly skills include:
+* javascript
+* html
+* Git.
```
Comment (codeschool-kiddo) [inline introduction.md]: Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

**Comment by sunchron** [comment]
I will make the change to skills.

**Comment by sunchron** [comment]
checkout for tag

**Pull request merged**